Compilation-Debian,-Ubuntu,-and-derivatives.md

```diff
@@ -21,16 +21,16 @@ sudo apt install build-essential cmake git ninja-build pkg-config libboost-dev l
 
 qBittorrent uses the Qt framework as the basis for its GUI.
 
-- qBittorrent 4.4.x requires at least Qt 5.15.2.
-- At the time of writing, the current `master` branch requires at least Qt 5.15.2.
+- qBittorrent 4.4.x requires at least Qt 6.5.0.
```

Comment:
If you really meant 4.4.x then this is false 
https://github.com/qbittorrent/qBittorrent/blob/1b0fd456439e5ce461189c6234b10b52b98d88f8/CMakeLists.txt#L15-L16